ShortenLink/Sources/Application/AppDelegate.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  AppDelegate.swift
+//  ShortenLink
+//
+//  Created by Phong Vo on 12/11/2021.
+//
+
+import Cocoa
+
+@main
+class AppDelegate: NSObject, NSApplicationDelegate {
+
+    private let statusItem = NSStatusBar.system.statusItem(withLength:NSStatusItem.squareLength)
+    private let popOver = NSPopover()
+
+    func applicationDidFinishLaunching(_ aNotification: Notification) {
+        setUpStatusBarIcon()
+        setUpPopOver()
+    }
+
+    func applicationWillTerminate(_ aNotification: Notification) {
+        // Insert code here to tear down your application
+    }
+}
+
+extension AppDelegate {
+
+    private func setUpStatusBarIcon() {
+        if let button = statusItem.button {
+            button.image = NSImage(named:NSImage.Name("status_bar_icon"))
```

Comment:
```suggestion
            button.image = NSImage(named: "status_bar_icon")
```
but later on we can use swiftgen.